sources/Coturn_config_rotate.sh

```diff
@@ -25,4 +25,8 @@ fi
 
 ynh_replace_string "^external-ip=.*\$" "$external_IP_line" "/etc/matrix-$app_instance/coturn.conf"
 
-exit 0
\ No newline at end of file
+setfacl -R -m user:turnserver:rX  /etc/matrix-$app_instance
+
+systemctl restart coturn-$app_instance.service
```

Comment:
Is it really needed to restart the service each time ? Just for the permission on the file ?

Reply:
Well, It was a mistake to not restart the service.

The idea here is just to restart the service while we update the config

Reply:
OK, got it.

Is a reload not enough ?
And, could you please remind me when this script is used ?

Reply:
> Is a reload not enough ?

It's not possible with this service

> And, could you please remind me when this script is used ?

As documented [here](https://github.com/YunoHost-Apps/synapse_ynh/blob/master/README.md#turnserver), it's used to update the public IP in the turnserver config.

Reply:
So that means that the service will be restart every 15 minutes. is it harmless for the users connected to the service ?

Reply:
Yes, it's only for user which as not a static IP. I think 1 or 2 hours could be also good. Anyway, I think there are not many solution if we need to keep up to date the real public IP in the turnserver config.

Reply:
So I guess the answer is no then. If it's harmless and without consequences, then ok.
It was just to now and be sure about it.